src/base/static/js/utils.js

```diff
@@ -154,7 +158,7 @@ var self = (module.exports = {
     collectionsSet,
     args,
     mapConfig,
-    callbacks,
+    callbacks
```

Comment:
I'm a bit confused by these removed commas, as it seems we use trailing commas most other places.

Reply:
I think Prettier distinguishes between lists of arguments to functions (where no trailing commas are expected), and other situations (where trailing commas are enforced).

Reply:
Note that we have Prettier configured in our `package.json` here: https://github.com/mapseed/platform/blob/master/package.json#L31

If using Prettier in your editor, you should be able to configure it by adding a `.prettierrc` file in the project root, which would look like this to match our `package.json`:

```
{
  "trailingComma": "all"
}
```

I like always adding the trailing comma because it reduces diffs, and makes it easier to move lines around. But I'm definitely open to try out other configurations as well.

We should check in a `.pretterrc` file, and perhaps have `package.json` read from it, then re-run Prettier on the whole codebase. Although I was waiting to get the `react-port` feature branch merged before doing that.

Reply:
I think when I switched editors my prettier settings got changed slightly-- I noticed I have trailing commas set to `es5` instead of `all`.

+1 for a `.prettierrc`